pkgs/development/libraries/science/biology/nifticlib/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchurl, cmake, zlib }:
+
+stdenv.mkDerivation rec {
+  pname    = "nifticlib";
+  pversion = "2.0.0";
+  name  = "${pname}-${pversion}";
+
+  src = fetchurl {
+    url    = "https://downloads.sourceforge.net/project/${pname}/${pname}/${pname}_2_0_0/${name}.tar.gz";
+    sha256 = "123z9bwzgin5y8gi5ni8j217k7n683whjsvg0lrpii9flgk8isd3";
```

Comment:
I get a wrong hash here:

```console
$ nix-shell --keep-going -j4 --option build-use-sandbox true -p nifticlib
these derivations will be built:
  /nix/store/wz3n6j1vpvd9ghg7fkr05g1pryc1b02n-nifticlib-2.0.0.tar.gz.drv
  /nix/store/71bin3yc2c6labb8b28cl89c56xnpm51-nifticlib-2.0.0.drv
building path(s) ‘/nix/store/c410n034lzs8jk8fl8fck3898x97j784-nifticlib-2.0.0.tar.gz’
trying https://downloads.sourceforge.net/project/nifticlib/nifticlib/nifticlib_2_0_0/nifticlib-2.0.0.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0   280    0     0    0     0      0      0 --:--:--  0:00:02 --:--:--     0
  0   274    0     0    0     0      0      0 --:--:--  0:00:03 --:--:--     0
  0   276    0     0    0     0      0      0 --:--:--  0:00:04 --:--:--     0
100 49088  100 49088    0     0   9817      0  0:00:05  0:00:05 --:--:--  236k
output path ‘/nix/store/c410n034lzs8jk8fl8fck3898x97j784-nifticlib-2.0.0.tar.gz’ has sha256 hash ‘1v9rrq9ccqbz904p3ypp0ny79djr35mlx17a4nvfkjvam1aixzis’ when ‘123z9bwzgin5y8gi5ni8j217k7n683whjsvg0lrpii9flgk8isd3’ was expected
cannot build derivation ‘/nix/store/71bin3yc2c6labb8b28cl89c56xnpm51-nifticlib-2.0.0.drv’: 1 dependencies couldn't be built
error: build of ‘/nix/store/71bin3yc2c6labb8b28cl89c56xnpm51-nifticlib-2.0.0.drv’ failed
/run/current-system/sw/bin/nix-shell: failed to build all dependencies
```

Did you forgot to update it?

Reply:
Looks like Sourceforge isn't giving the same file each time ...

Reply:
Even the file size is changing.  Looks like Sourceforge is broken at the moment.

Reply:
I think your download link is should be different. It currently returns dynamic html. We also use
urls like `url = "mirror://sourceforge/statifier/statifier-${version}.tar.gz";` for sourceforge to fallback to mirrors automatically.

Reply:
@Mic92 Typo in the URL, sorry ... fixed now.